### DISCUSSION TITLE
Adding Genbank Accession No. to GeOMe

### DISCUSSION THREAD

**Discussion by steierj** [steierj/ReadtheDocsFork#7] [Ideas]
Are we making this an official part of our standard protocols? Where should it be added? "Populating GeOMe FIMS spreadsheet" page in column definitions, and add a reminder to add this data to the FIMS at the end of the Genbank submission pages. (Geneious and Sub Portal both)

**Comment by steierj** [comment]
Also update the Genbank_update page text regarding Genbank accessions not being in the FIMS.

**Reply by redmondn** [reply]
We should have an answer to this in a few weeks after chatting to Chris.
